Author: Daniel Kahn Gillmor <dkg@fifthhorseman.net>
Description: Avoid trying to build anything for ios, macos, or windows.  Debian doesn't need it, and we don't necessarily have straightforward access to the dependencies

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -32,11 +32,6 @@
 license = "MIT OR Apache-2.0"
 repository = "https://github.com/hwchen/keyring-rs.git"
 
-[[example]]
-name = "iostest"
-crate-type = ["staticlib"]
-path = "examples/ios.rs"
-
 [dependencies.lazy_static]
 version = "1"
 
@@ -72,32 +67,18 @@
     "platform-linux",
     "platform-freebsd",
     "platform-openbsd",
-    "platform-macos",
-    "platform-ios",
-    "platform-windows",
 ]
 platform-freebsd = ["linux-secret-service"]
-platform-ios = ["security-framework"]
 platform-linux = [
     "linux-secret-service",
     "linux-keyutils",
 ]
-platform-macos = ["security-framework"]
 platform-openbsd = ["linux-secret-service"]
-platform-windows = [
-    "windows-sys",
-    "byteorder",
-]
-windows-test-threading = []
 
 [target."cfg(target_os = \"freebsd\")".dependencies.secret-service]
 version = "3"
 optional = true
 
-[target."cfg(target_os = \"ios\")".dependencies.security-framework]
-version = "2.6"
-optional = true
-
 [target."cfg(target_os = \"linux\")".dependencies.linux-keyutils]
 version = "0.2"
 features = ["std"]
@@ -107,22 +88,6 @@
 version = "3"
 optional = true
 
-[target."cfg(target_os = \"macos\")".dependencies.security-framework]
-version = "2.6"
-optional = true
-
 [target."cfg(target_os = \"openbsd\")".dependencies.secret-service]
 version = "3"
 optional = true
-
-[target."cfg(target_os = \"windows\")".dependencies.byteorder]
-version = "1.2"
-optional = true
-
-[target."cfg(target_os = \"windows\")".dependencies.windows-sys]
-version = "0.52"
-features = [
-    "Win32_Foundation",
-    "Win32_Security_Credentials",
-]
-optional = true
